packages/restapi/src/lib/user/profile.updateUser.ts

```diff
@@ -25,9 +25,10 @@ type ProfileUpdateProps = {
    *  Profile properties that can be changed
    */
   profile: {
-    name?: string;
-    desc?: string;
-    picture?: string;
+    name?: string | null;
```

Comment:
`null` should not be there as a type
Reason - we don't want to allow the user to pass name or desc as null.

Reply:
the params are already kept as optional to give the idea that if a user has not passed it, then that value is not updated